src/helpers/moderation-status.ts

```diff
@@ -1,5 +1,28 @@
+import type { IconComponent } from '@hypothesis/frontend-shared';
+import {
+  CautionIcon,
+  CheckAllIcon,
+  DottedCircleIcon,
+  RestrictedIcon,
+} from '@hypothesis/frontend-shared';
+
 export type ModerationStatus = 'PENDING' | 'APPROVED' | 'DENIED' | 'SPAM';
 
+/**
+ * Map of moderation statuses to their corresponding icon
+ */
+export const moderationStatusToIcon: Record<ModerationStatus, IconComponent> = {
+  PENDING: DottedCircleIcon,
+  APPROVED: CheckAllIcon,
+  DENIED: RestrictedIcon,
+  SPAM: CautionIcon,
+} as const;
```

Comment:
Do we need two separate records or could we just use one `Record<ModerationStatus, { label: string, icon: IconComponent }>` map, with callers using `moderationStatusInfo[status].{label, icon}` as needed?

Reply:
Yeah, that actually makes sense